### ISSUE TITLE
Seal of righteousness

### ISSUE THREAD

**Issue by zikka** [RetroWoW/RetroWoW#176]
**Description**: Seal of righteousness does not seem to benefit from spell power gear at all. the judgement does. also, it seems to get the holy damage bonus from judgement of the crusader, just not gear piece spell power.

**Current behavior**: on a character with around 300 spell power, seal of righteousness does base damage, if i judge crusader first, it benefits from that but not my gear's spell power.

**Expected behavior**: righteousness should have a spell power coefficient that varies by weapon speed

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Source:**

**Comment by zikka** [comment]
this is a pretty important issue because without spell power scaling righteousness is useless. When playing a caster paladin that does not use seal of command, this one of our main forms of damage. thanks

**Comment by modnaros** [comment]
Seal of Command doesn't properly scale w/ sp either to my knowledge (the judgement not the proc)

**Comment by zikka** [comment]
i have been using it in place of righteousness.... which is terrible for my build but better than nothing. the judgement of command seems to scale about right. although i haven't tested it. do you mean it is too high or too low? i believe it should get about 40% of your spell power on the judge, and the proc should have 29%

**Comment by modnaros** [comment]
I believe the judgement of command scaling is a little too low.

**Comment by zikka** [comment]
yes you're right. it should be 43% or so and it seems to be about 29%. maybe they just put 29% for both.

**Comment by zikka** [comment]
do they test it or do we have to?

**Comment by zikka** [comment]
I'd love to demonstrate this in game if someone wanted to work with me.  This seal really needs to work for all paladin specs to be playable.

**Comment by hollowtipz2123** [comment]
@Bam27 Is there any word on updating SoR to scale with spell damage? In addition, can we confirm that SoR hits are counted as melee hits?
